packages/kotti-table/src/logic/order.js

```diff
@@ -24,6 +24,7 @@ export const mutations = {
 			col.order = index
 			col.index = index
 		})
+		//FIXME: there is an issue in the line below
```

Comment:
Please describe the issue

Reply:
fixed it - was a function that returned nothing --> @carsoli

Reply:
i don't see the fix but it's a missing return statement in the exported function, not the getter
should work on issues of the table again soon (next tech day probably) so we can report this as an issue now since it's out of this scope @Goldziher